src/debug/ee/i386/dbghelpers.S

```diff
@@ -0,0 +1,69 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include "unixasmmacros.inc"
+
```

Comment:
Is it possible to use `.intel_syntax noprefix` for x86 like we do for AMD64 instead of using the default GAS syntax? It would be highly preferrable.